test/controllers/callbacks_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
```

Comment:
you can remove all those autogenerated test and fixture files. Since you have `rspec` and `factory_bot`, I doubt that you are going to use this type of test and fixture files

Reply:
removed all autogenerated files from a repository